h2o-py/h2o/explanation/_explain.py

```diff
@@ -1414,6 +1437,12 @@ def _get_response(mean_response, show_logodds):
     else:
         return mean_response
 
+def _get_stddev_response(stdev_response, mean_response, show_logodds):
+    if show_logodds:
+        return 1 / np.sqrt(len(mean_response) * mean_response * (1 - mean_response))
```

Comment:
https://stats.stackexchange.com/questions/266098/how-do-i-calculate-the-standard-deviation-of-the-log-odds
https://stats.stackexchange.com/questions/420795/log-odds-standard-error